pastehunter/outputs/http_output.py

```diff
@@ -0,0 +1,38 @@
+import logging
+import requests
+import json
+from common import parse_config
+
+logger = logging.getLogger('pastehunter')
+
+config = parse_config()
+
+class HttpOutput():
+    def __init__(self):
+        self.valid = True
+        self.endpoint_url = config['outputs']['http_output']['endpoint_url']
+        self.http_auth = config['outputs']['http_output']['http_auth']
+        self.http_user = config['outputs']['http_output']['http_user']
+        self.http_password = config['outputs']['http_output']['http_password']
+
+        if self.endpoint_url == '':
+            logging.error("HTTP endpoint not configured")
+            self.valid = False
+
+    def store_paste(self, paste_data):
+        if self.valid:
+            
+            json_data = paste_data
+            del json_data['raw_paste']
```

Comment:
This seems like something that should be configurable and not enabled by default (maybe `ignore_fields` in the http config)?

Reply:
your right, i just added it